src/pages/api/revalidate.ts

```diff
@@ -0,0 +1,82 @@
+import type { NextApiRequest, NextApiResponse } from 'next/types'
+import { StatusCodes } from 'http-status-codes'
+import { validateToken } from 'lib/api-validate-token'
+import { cachedGetProductData } from 'lib/get-product-data'
+
+/**
+ * Accepts a POST request with a product slug, triggers revalidation for all of a product's paths
+ * specified in its latest nav data.
+ */
+async function handler(request: NextApiRequest, response: NextApiResponse) {
```

Comment:
Would we also have a unique `revalidate` handler for tutorials that only handles a single tutorial or collection path?

Reply:
I'll probably update this handler to also handle an arbitrary list of paths to revalidate 👍